docs/configuration/http_conn_man/route_config/rate_limits.rst

```diff
@@ -25,14 +24,10 @@ stage
 kill_switch_key
   *(optional, string)* The key to be set in runtime to disable this rate limit configuration.
 
-route_key
-  *(optional, string)* Specifies a descriptor value to be used when rate limiting for a route.
-  This information is used by the actions if it is set.
-
 actions
   *(required, array)* A list of actions that are to be applied for this rate limit configuration.
-  Order matters as the actions are processed sequentially and the descriptors will be composed in
-  that sequence.
+  Order matters as the actions are processed sequentially and the descriptor will be composed by
+  appending decriptor entries in that sequence.
```

Comment:
at this point I think it would be useful to provide a small example in the docs (kind of like we do for traffic shifting).